BUILD

```diff
@@ -2308,4 +2308,29 @@ grpc_cc_library(
     ],
 )
 
+#TODO: Get this into build.yaml once we start using it.
+grpc_cc_library(
+    name = "google_protobuf",
```

Comment:
This name is colliding with some internals. Please change it to "google_upb" for example.

Reply:
changed to google_protobuf_upb